drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -118,12 +118,26 @@ local function find_child(parent, ep_id)
   return parent:get_child_by_parent_assigned_key(string.format("%02d", ep_id))
 end
 
+local function detect_bridge(device)
+  for _, ep in ipairs(device.endpoints) do
+    for _, dt in ipairs(ep.device_types) do
+      if dt.device_type_id == 0x000E then -- look for aggregator type
```

Comment:
Do we have the device types available in a lib already? If not, I'd suggest adding a local var to define the device type so it's less of a magic number

Reply:
Done!